### DISCUSSION TITLE
Recognizing Repositories’ Role Models ⭐  [Repositories Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#128859] [Repositories]
![Header for the Repositories category shows a dark purple abstract background with white text reading “Repositories Check-in”. In the foreground there is an image of a person holding a magnifying glass next to a cartoon representation of a group of repositories. To the right, there is a GitHub logo in two-tone purple.](https://github.com/community/community/assets/144172738/06f2261b-e8f4-40a3-b92b-99cf29b86134)

This month we’re doing things a bit differently in this category. Our members are what make this space so great and I want to dedicate some time to share who has made our category so great and how you can follow their footsteps. 

## Wall of Fame 🤩
* @pedrolemos4: Helped a user enable Code Scanning in their Repository Settings by sharing documentation in [their reply](https://github.com/orgs/community/discussions/126642). They didn’t need to recreate the wheel or tell the member where to search, but by directly linking them to our Docs they ensured the OP and future askers of this questions had GitHub’s trusted steps to get this enabled. 
* @MostlyKIGuess: Wrote a [stellar response](https://github.com/orgs/community/discussions/123235) for duplicating a repository through GitHub’s interface! Their response contained two different options to answer the question which showcased their expertise as a developer and allowed the OP and other users to use the option that worked best for them 💪. And both options were great! They included clear steps, call outs to where buttons may be located, and the Git commands needed and clearly labeled. This answer is so well-written as it is helpful to anyone regardless of their coding background. Well done! 
* @electrofelix: Shared amazing [feedback](https://github.com/orgs/community/discussions/118843#discussioncomment-9571204) on our Push rule private discussion. It was detailed and specific - they shared exactly what they were trying to achieve, what they were encountering, included workarounds they considered, and clearly articulated what their use case was. Bonus: They were responsive to our Product Manager who had follow up questions as well. Thank you for helping us improve @electrofelix! 
            

## Being a great community citizen 🌏

How you can help others, further develop your own skills, and even get recognized yourself! 

* **Have the Heart of a Teacher** 🧑‍🏫: Kindness, compassion, inspiration, and the joy of learning
   * We may not have all the answers, but we have a positive mindset and a desire to build and help others. 
   * I appreciate how upbeat @CopperEagle was in [their comment](https://github.com/orgs/community/discussions/124182#discussioncomment-9463983) and the great screenshots they used to explain the answer. 
* **Be Authentic** 💟: Verify what you are sharing and take personal responsibility for what you’re posting 
    * Read and revise the content before you post it, do not just post AI-generated content verbatim to inflate your reputation or give a false impression of product expertise.
     * Not only do they remind other users to be authentic, but @markolalovic also shared notes and another Discussion in [their response](https://github.com/orgs/community/discussions/122143#discussioncomment-9305220) for a robust response.  
* **Don’t Assume** 😀: The GitHub Community is made up of users all over the world and of varying abilities and lived experiences 
   * Approach other users with curiosity and an open mind, we can all learn from one another! 
   * This [comment](https://github.com/orgs/community/discussions/125656) from @davevad93 was a great example of this as he included background information and didn’t assume any prior knowledge making this response useful for anyone! Also kudos to linking to further information as well 👏. 
* **Ask Questions** ❓: What have they tried already? Which version are they using? Do they have any screenshots? 
   * Asking Questions encourages folks to add more information which makes it more likely they’ll get a response.
    * @gurjeetsinghvirdee did just that when trying to assist [here](https://github.com/orgs/community/discussions/122653). 
* **Use Links** 🔗: Documentation, YouTube, a helpful project hosted on GitHub that may be a workaround or answer their questions, be sure to share any context or additional knowledge too! 
   * We have so much information at our disposal, by sharing what we’ve found to work circulates quality content and increases all our knowledge. 
   * @spenserblack included two links in [their reply](https://github.com/orgs/community/discussions/122433). One link was explaining what submodules are and they included their own summary. Then what really took this to the next level, was they also used a link to showcase an example of what a submodules file will look like. Bravo! 

A huge thank you to @spenserblack, @gurjeetsinghvirdee, @davevad93, @markolalovic, @CopperEagle, @electrofelix, @MostlyKIGuess, and @pedrolemos4 for modeling amazing community behavior 🙇. 

Don’t forget, for community members who post questions, [marking responses that solves your question as the answer](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer) goes a long way. When you mark a comment in the Discussion as an answer, GitHub will highlight the comment to help other community members quickly find the answer as well.

![The image is a screenshot showing where users can click to mark a response as the answer](https://github.com/user-attachments/assets/d2e6fb39-907f-4630-9030-f2dc4b44be6f)

Remember to thank these and other community members that take the time to comment and help troubleshoot these problems. An acknowledgement of a job well done can have an incredible effect and make someone’s day. ☀️ We’re here to learn, discover and connect so try and mark a comment as an answer if suitable, give a thank you and select a reaction as a way to show your appreciation!

Celebrate the above users by giving them some love below!

**Comment by MostlyKIGuess** [comment]
Lol idk how I got selected, but thank you :3
I am just giving it back .

**Reply by queenofcorgis** [reply]
@MostlyKIGuess you got selected by doing great work in the community - thank you!

**Comment by davevad93** [comment]
Thank you @queenofcorgis , I'm so glad to be part of this community!

**Reply by queenofcorgis** [reply]
@davevad93 Thank you and we are so glad to have you 💟

**Comment by CopperEagle** [comment]
Thank you very much @queenofcorgis :heart:

(Currently studying for exams so I saw the mention a bit late ;)

**Reply by queenofcorgis** [reply]
@CopperEagle hope your exams went well!

**Reply by CopperEagle** [reply]
@queenofcorgis I can't tell yet, I have the exams at the end of summer :) In any case, thanks for your mention :heart:

**Reply by queenofcorgis** [reply]
@CopperEagle in that case best of luck!

**Comment by queenofcorgis** [comment]
Thank you so much to June's outstanding contributors, we're celebrating some more in [July's repositories check-in](https://github.com/orgs/community/discussions/134083).
